src/gui/modals/EffectSelectDialog.cpp

```diff
@@ -89,191 +89,232 @@ EffectSelectDialog::EffectSelectDialog( QWidget * _parent ) :
 			name = it->desc->displayName;
 			type = "LMMS";
 		}
-		m_sourceModel.setItem( row, 0, new QStandardItem( name ) );
-		m_sourceModel.setItem( row, 1, new QStandardItem( type ) );
+		m_sourceModel.setItem(row, 0, new QStandardItem(name));
+		m_sourceModel.setItem(row, 1, new QStandardItem(type));
 		++row;
 	}
 
-	// setup filtering
-	m_model.setSourceModel( &m_sourceModel );
-	m_model.setFilterCaseSensitivity( Qt::CaseInsensitive );
+	// Setup filtering
+	m_model.setSourceModel(&m_sourceModel);
+	m_model.setFilterCaseSensitivity(Qt::CaseInsensitive);
+
+	QHBoxLayout* mainLayout = new QHBoxLayout(this);
 
-	ui->filterEdit->setPlaceholderText(tr("Search"));
-	ui->filterEdit->setClearButtonEnabled(true);
-	ui->filterEdit->addAction(embed::getIconPixmap("zoom"), QLineEdit::LeadingPosition);
+	QVBoxLayout* leftSectionLayout = new QVBoxLayout();
 
-	connect(ui->filterEdit, &QLineEdit::textChanged, &m_model, &QSortFilterProxyModel::setFilterFixedString);
-	connect(ui->filterEdit, &QLineEdit::textChanged, this, &EffectSelectDialog::updateSelection);
-	connect(ui->filterEdit, &QLineEdit::textChanged, this, &EffectSelectDialog::sortAgain);
+	QStringList buttonLabels = { "All", "LMMS", "LADSPA", "LV2", "VST" };
+	for (const QString& label : buttonLabels)
+	{
+		QPushButton* button = new QPushButton(label, this);
+		button->setFixedSize(50, 50);
+		button->setFont(QFont("Arial", 15, QFont::Bold));
```

Comment:
Why Arial, and not the system font? We rarely specify a particular font elsewhere, so this may look inconsistent.